source/WorkingPractices/branch_migration.rst

```diff
@@ -3,6 +3,15 @@
 Migrating a branch from fcm to git
 ==================================
 
+.. important::
+
+    If migrating an old trac  ticket, please make sure to,
+
+    * Link from the trac ticket to the new PR/Issue
+    * Link from the new PR/Issue back to the trac ticket
```

Comment:
Perhaps combine these in a single bullet point: cross-link tack ticket and GitHub Issue/PR?